test/e2e/multi-cluster-deployment/use-kubestellar.sh

```diff
@@ -28,7 +28,9 @@ metadata:
   name: nginx-bindingpolicy
 spec:
   clusterSelectors:
-  - matchLabels: {"location-group":"edge"}
+  - matchLabels: {"name":"cluster1"}
+  - matchExpressions:
+    - { key: name, operator: In, values: [cluster2, cluster3] }
```

Comment:
I understand the reason for changing this file (you want to show an example how to use it and also verify this is tested), but not sure I would do this change.
few reasons:
- there is no `cluster3` in the e2e test.
- this change is testing the introduced change, but removes the test that matchLabels can select more than one cluster.
- it might be better to put this example in a README file. we can add a section that shows examples for how to select clusters.

Reply:
I agree on cluster3, removed while updating on slack.

As for the 2nd point, what do you think about testing both here? As in:
- Revert the test to how it was
- After success: change the BP to select no clusters
- Wait until update is reflected in cluster
- Change the BP to select the two clusters with the selectors I wrote
- Test